GapBackfill/local.settings.json

```diff
@@ -1,15 +1,15 @@
 {
   "IsEncrypted": false,
   "Values": {
-    "AzureWebJobsStorage": "[Add your connection string here]",
```

Comment:
I prefered to have a value that indicates "Add your con string here"...don´t you like it :-)?

Reply:
ohh, after removing my connection string, I forgot to add that message back, let me update!